src/python/pants/core/goals/package.py

```diff
@@ -51,11 +51,13 @@ class BuiltPackage:
 class OutputPathField(StringField, AsyncFieldMixin):
     alias = "output_path"
     help = (
```

Comment:
No changes to this file. Only fixed it so each paragraph is a distinct code block.